supervisely/project/project_settings.py

```diff
@@ -102,28 +113,40 @@ def __init__(
         multiview_tag_name: Optional[str] = None,
         multiview_tag_id: Optional[int] = None,
         multiview_is_synced: bool = False,
+        labeling_interface: Optional[LabelingInterface] = None,
     ):
         self.multiview_enabled = multiview_enabled
         self.multiview_tag_name = multiview_tag_name
         self.multiview_tag_id = multiview_tag_id
         self.multiview_is_synced = multiview_is_synced
 
+        if labeling_interface is not None:
+            labeling_interface = str(labeling_interface)
+        self.labeling_interface = labeling_interface
+
         if multiview_enabled is False and multiview_is_synced is True:
             logger.warn(
                 "The 'Group Images sync mode' is enabled, but it won't effect while multi-view mode is disabled. Please enable the multi-view mode (a.k.a. 'Group Images mode')."
             )
 
+        if labeling_interface is not None and labeling_interface not in LabelingInterface.values():
```

Comment:
move second condition `labeling_interface not in LabelingInterface.values()` to 124-125 lines